src/Icons/Icons.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import trashUrl, { ReactComponent as TrashCan } from './icons/trash-can.svg';
```

Comment:
What are you using the URLs for in this file?

Reply:
We are not at the moment, but that lib that compiles the code exposes them to us, and I thought it would be good pass them through to the app.